config/database.yml

```diff
@@ -21,6 +21,8 @@ default: &default
   # For details on connection pooling, see rails configuration guide
   # http://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  username: postgres
+  password: io
```

Comment:
crap -- I didn't realize it wasn't in the `.gitignore` anymore :( Sorry bout that -- The copy of `database.yml` I had in my dev env at home had nothing for user/pass after I syncd. (It was a couple weeks behind)

What's the best way to do this? IIRC Sean had to remove it from `.gitignore` because of TravisCI -- I've not had to deal with that before so I'm not sure what's best.